vmware-base/src/com/cloud/hypervisor/vmware/mo/SnapshotDescriptor.java

```diff
@@ -166,12 +168,7 @@ public void removeDiskReferenceFromSnapshot(String diskFileName) {
             assert (false);
             s_logger.error("Unexpected exception ", e);
         } finally {
-            if (out != null) {
-                try {
-                    out.close();
-                } catch (IOException e) {
-                }
-            }
+            closeAutoCloseable(out, "buffered write for bytestream");
```

Comment:
Why not use the try-with-resource syntax instead of doing all this? IIRC you had submitted earlier commits with similar changes.